docs/multicluster/getting-started.md

```diff
@@ -0,0 +1,417 @@
+
+   * For any given release `<TAG>` (e.g. `v1.5.0`).
+
+     ```bash
+     kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-multicluster-leader-global.yml
+     ```
+
+   * To deploy the latest version (checkout and built from the `feature/multi-cluster` branch).
+
+     ```bash
+     kubectl apply -f multicluster/build/yamls/antrea-multicluster-leader-global.yml
+     ```
+
+2. Install multi-cluster controller in the leader cluster, since multi-cluster controller is
+running as namespaced deployment, you should create a namespace first, then apply the
+manifest with new namespace. below are sample commands.
+
+* For any given release `<TAG>` (e.g. `v1.5.0`).
+  
+  ```bash
+  kubectl create ns antrea-mcs-ns
+  curl -L https://github.com/antrea-io/antrea/releases/download/v1.5.0/antrea-multicluster-leader-namespaced.yml >   antrea-multicluster-leader-namespaced.yml
+  sed 's/changeme/antrea-mcs-ns/g' antrea-multicluster-leader-namespaced.yml | kubectl apply -f -
+  ```
+  
+* To deploy the latest version (checkout and built from the `feature/multi-cluster` branch).
+  
+  ```bash
+  kubectl create ns antrea-mcs-ns
+  multicluster/hack/generate-manifest.sh -l antrea-mcs-ns | kubectl apply -f -
+  ```
+  
+### Installation in the Member Cluster
+
+You can run below commands to install multi-cluster controller to all
+member clusters, it will install a member controller in the namespace `kube-system`
+by default.
+
+* For any given release `<TAG>` (e.g. `v1.5.0`).
+
+```bash
+kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-multicluster-member.yml
+```
+
+* To deploy the latest version (checkout and built from the `feature/multi-cluster` branch).
+
+```bash
+kubectl apply -f multicluster/build/yamls/antrea-multicluster-member.yml
+```
+
+### Installation in a Shared Cluster
+
+There is no deployment dependency between member and leader clusters if you are using a cluster
+as a dedicated leader cluster. But if you'd like to run both leader and member controllers in
+one cluster, below is the required deployment sequence:
+
+1. Follow the step in section [Installation in the Member Cluster](#installation-in-the-member-cluster)
+to install member controller first.
+2. Follow the step 2 only in section [Installation in Dedicated Leader Cluster](#installation-in-dedicated-leader-cluster)
+to install leader controller. the global CRDs have been installed when you deploy the member controller.
+
+## ClusterSet
+
+In an Antrea multi-cluster cluster set, there will be at least one leader cluster
+and two member clusters. In the below examples we will use cluster set id `test-clusterset` which
+has two member clusters with cluster id `test-cluster-east`, `test-cluster-west` and one
+leader cluster with id `test-cluster-north`.
+
+### Setting up Access to Leader Cluster
+
+We first need to setup access from all member clusters into the leader cluster's API server.
```

Comment:
set up

into -> to

Maybe access to the leader cluster's API server for all member clusters